zerver/webhooks/clickup/doc.md

```diff
@@ -0,0 +1,56 @@
+# Zulip ClickUp integration
+!!! tip ""
+
+    Note that [Zapier][1] is usually a simpler way to
+    integrate ClickUp with Zulip.
+
+Get Zulip notifications for your ClickUp space!
+
+[1]: ./zapier
+{start_tabs}
+1. {!create-stream.md!}
+
+1. {!create-an-incoming-webhook.md!}
+
+1. {!generate-webhook-url-basic.md!}
+
+1. Download [zulip-clickup.py][2]. `Ctrl+s` or `Cmd+s` on that page should
+   work in most browsers.
+
+1. Make sure you have a working copy of [Python](https://realpython.com/installing-python/), you will need it to run the script later.
+
+1. Collect your ClickUp **Team ID**:
+    * Go to your ClickUp home. The URL should look like `https://app.clickup.com/<TEAM_ID>/home`. Note down the
+    `<TEAM_ID>`.
+
+
+1. Collect your ClickUp **Client ID & Client Secret** :
+    1.  Go to <https://app.clickup.com/settings/team/clickup-api> and click **Create an App** button.
```

Comment:
Yeah, I agree with you here. But, it seems like there's a bug when using different types of lists (ordered & unordered) inside the {start_tabs}/{end_tabs} block. 

Here's what is rendered when I try to use the * unordered list:
<details>

![image](https://github.com/zulip/zulip/assets/108744694/9c46e486-45f1-47d5-8b28-48ca4d94f8d2)
![image](https://github.com/zulip/zulip/assets/108744694/1631fe77-1735-4053-8871-59c66523d371)
using the - unordered list yields the same result

</details>

Using unordered list outside and inside the block seems to change all other list type inside the block into unordered list: 
<details>

![image](https://github.com/zulip/zulip/assets/108744694/5045d9f0-c559-4b25-b704-c45dd5c3fc56)
![image](https://github.com/zulip/zulip/assets/108744694/94021325-40d6-4cc2-b0e4-fa11765ec333)

</details>

The best I could think of it to "manually" use bolded hyphens to act as the unordered list:
<details>

![image](https://github.com/zulip/zulip/assets/108744694/274a7187-ddf4-429f-9e21-37415632ae22)
![image](https://github.com/zulip/zulip/assets/108744694/979b7746-f65d-4b65-ab48-ba16de2c8c0d)

</details>

Reply:
Ah! Okay. I removed a CSS rule recently from `integrations.css` that seems to be coming in play here. We need to add a rule for unordered list elements inside an ordered list element.

@karlstolley mentioning you here to see if my suggestion below seems like the correct fix here.

@PieterCK - You could do this update as a separate prep-commit here ... or maybe a separate PR would be better. Because it would be good to fix this sooner rather than later and I'm not sure where in the review process the overall new ClickUp integration is.

```scss
.portico-landing.integrations {
...
    & ol li {
        list-style: none;

        & ul li {
            list-style: inherit;
        }
    }
```

test in airbrake doc

```markdown
1. {!generate-webhook-url-basic.md!}

    * test one
    * two test
    * test three
```

With the CSS changes that renders as ...

<details>
<summary>Test screenshot</summary>

*Note the unordered list element at the bottom is rendered as expected as well.*
![Screenshot from 2024-04-10 16-00-06](https://github.com/zulip/zulip/assets/63245456/72efac9f-d855-4f49-9eb2-bcc16317f16a)
</details>

Reply:
noted, I'll shoot up a new PR for the `integrations.css` fix

Reply:
tested it with ClickUp doc, much nicer :)

![image](https://github.com/zulip/zulip/assets/108744694/a1fbf348-19d0-416b-b764-8370463a3441)

Reply:
created a new issue just to track the PR, #29669 
and the PR for the css fix: #29671
@laurynmm